### PR TITLE
30 sp animals

### PR DESCRIPTION
**Pull request by potnurusrilatha**
added all animals in to js please and review it

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    padding: 0px;
+    list-style-type: none;
+}
+
+.sidebar ul li {
+    text-align: center;
+    padding: 8px;
+    font-size: 22px;
+    cursor: pointer; 
+    color: #211000;
```
Comment (TobiasKjernell) [inline styles/index.css]: here we can use the variable, var(--textcolor-main);
To use it, make sure you link the variable stylesheet in html.

File: index.html [hunk 2]
```diff
@@ -3,9 +3,23 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Homepage</title>
```
Comment (TobiasKjernell) [inline index.html]: Think here we can have the title "Home"

File: styles/index.css (hunk 1 above)
Comment (potnurusrilatha) [reply styles/index.css]: ok

**Review by jasminedns — APPROVED** [review]
for me it's good to go!

**Review by Moon2024-light — APPROVED** [review]
looks good !

**Pull request merged**